### PR TITLE
Rfa01c cleanup

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### Which User story does this relate to?
-
https://osi-cwds.atlassian.net/browse/CALS-4907

#### Brief feature description
- refactor of current rfa 01c card. 

TODO: 
- is the approach for how we will handle relationships settled?
- am i missing anything css wise? 
- up the test coverage

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
78.94% Statements
 64.11% Branches 
 72.99% Functions 
 79.83% Lines

### CODE REVIEW

**Comment by Nasir2712** [comment]
@lauherkcambria I tested the form and it saves everything. I think these are some of the changes that we need to address if we are going with the design document. 
i) Child's Name text should be changed to 'Name of the Child'
ii) the label is missing for the yes or no component. It just says YES or NO without any text on the top.
iii) css wise - Child's Name(Name of the Child), Child's Relationship to Applicant and Child's Education should be in the same line with Child Identification. I am not sure if we have to use custom stylings for this but just putting it out there.

File: app/javascript/components/common/pageHeader.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const PageHeader = ({headerLabel}) => (
+  <div className='col-xs-13 col-sm-13 col-md-13 col-lg-13'>
```
Comment (Nasir2712) [inline app/javascript/components/common/pageHeader.js]: I think this is a typo. Column width should be 12.

File: app/javascript/rfa_forms/rfa01c_edit_view/desiredChildCard.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    this.props.setParentState(index, 'school_address', schoolAddressObj.toJS())
+  }
+
+  render () {
+    const child = this.props.desiredChild
+    const index = this.props.index
+    return (
+      <div className='row'>
+        <div key={'desiredChildRemoveLink' + index} className='row'>
+          <a onClick={(event) => this.props.clickClose(index)} className='pull-right remove-btn'>Remove</a>
```
Comment (Nasir2712) [inline app/javascript/rfa_forms/rfa01c_edit_view/desiredChildCard.js]: can we use the remove link component here?

File: app/javascript/rfa_forms/rfa01c_edit_view/desiredChildDetails.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  suffixTypes: PropTypes.array,
+  genderTypes: PropTypes.array,
+  countyTypes: PropTypes.array
+}
+
+DesiredChildDetails.defaultProps = {
+  index: 0,
+  suffixTypes: [],
+  genderTypes: [],
+  countyTypes: []
```
Comment (Nasir2712) [inline app/javascript/rfa_forms/rfa01c_edit_view/desiredChildDetails.js]: can you add idPrefix also in default props?

File: app/javascript/rfa_forms/rfa01c_edit_view/desiredChildCard.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    )
+  }
+}
+
+DesiredChildCard.propTypes = {
+  index: PropTypes.number,
+  schoolGrades: PropTypes.array.isRequired,
+  countyTypes: PropTypes.array.isRequired,
+  suffixTypes: PropTypes.array.isRequired,
+  desiredChild: PropTypes.object.isRequired,
```
Comment (Nasir2712) [inline app/javascript/rfa_forms/rfa01c_edit_view/desiredChildCard.js]: idPrefix proptype is missing

File: app/javascript/rfa_forms/rfa01c_edit_view/index.js [hunk 5]
```diff
@@ -68,54 +70,42 @@ export default class Rfa01cList extends React.Component {
...
-                countyTypes={this.props.countyTypes}
-                validator={this.validator}
-                errors={this.state.errors.child}
-                nameSuffixTypes={this.props.nameSuffixTypes}/>
-            </div>
-          </div>
-        </div>
-      </div>
+      <PageTemplate
+        headerLabel='Resource Family Application - Confidential (RFA O1C)'
```
Comment (Nasir2712) [inline app/javascript/rfa_forms/rfa01c_edit_view/index.js]: 👍 looks neat :)

File: app/javascript/rfa_forms/rfa01c_edit_view/desiredChildCard.js (hunk 2 above)
Comment (lauherkcambria) [reply app/javascript/rfa_forms/rfa01c_edit_view/desiredChildCard.js]: as discussed on friday, we weren't able to get it to work. we can explore that refactor as a part of a reserah spike later if necessary.

**Pull request merged**